visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import execa from 'execa';
+import fse from 'fs-extra';
+
+// Main wrapper should be defined as a first one -
+// it will be used to generate golden screenshots,
+// rest of wrappers will be compared to it.
+const wrappers = ['js', 'angular-13', 'react'];
+// const wrappers = ['js', 'angular-13', 'react', 'vue'];
```

Comment:
If I understand correct, the array containing `vue` is temporarily commented out because of the problem with building the `vue` demo. (as discussed before)

I tried to manually recreate what the script is doing, and the `start` command for `vue/demo` fails with an error indeed. I think it has something to do with the example being a part of an npm workspace.

If I'm not mistaken, the `demo` example is looking for a dependency in a wrong place, because the `npm i` command has been run _inside_ of the example directory, not in the root of its "sub-workspace".

The examples were generally designed to be installed in bulk, [using the `examples:install` scripts in the root `package.json`](https://github.com/handsontable/handsontable/blob/12.2.0/package.json#L31). It basically propagates the `install` command to all of the "sub-workspaces", [like the one defined in `docs/vue/package.json`](https://github.com/handsontable/handsontable/blob/12.2.0/examples/next/docs/vue/package.json#L5-L7).

When running `npm install` in `next/docs/vue` and _then_ running `npm run start` inside the example directory, it started alright. 
That being said, installing each of the examples individually should work as well, so this problem probably needs some investigating.

Reply:
I've fixed that by implementing the `npm run examples:install next/visual-tests` script for Examples within https://github.com/handsontable/handsontable/pull/10165/commits/3a7954606ea62b9c9a3c9952b555634a1aa549be.